src/entity/ParkingPlace.ts

```diff
@@ -0,0 +1,67 @@
+import { Entity, Column, ManyToOne, JoinColumn } from 'typeorm';
+import Model from './Model';
+import { ParkingSector } from './ParkingSector';
+import { Camera } from './Camera';
+
+export enum ParkingPlaceType {
+  STANDARD = 'STANDARD',
+  STAFF = 'STAFF',
+  EMERGENCY = 'EMERGENCY',
+  ELECTRIC = 'ELECTRIC'
+}
+
+@Entity()
+export class ParkingPlace extends Model {
+  @Column({ length: 100, unique: true })
+  name: string;
+
+  @Column({ type: 'decimal' })
+  x1: number;
+
+  @Column({ type: 'decimal' })
+  y1: number;
+
+  @Column({ type: 'decimal' })
+  x2: number;
+
+  @Column({ type: 'decimal' })
+  y2: number;
+
+  @Column({ type: 'decimal' })
+  x3: number;
+
+  @Column({ type: 'decimal' })
+  y3: number;
+
+  @Column({ type: 'decimal' })
+  x4: number;
+
+  @Column({ type: 'decimal' })
+  y4: number;
```

Comment:
what are x1,x2,... for?

Reply:
@sanohin they are parking slot rectangle edges coordinates